webpack/assets/javascripts/react_app/components/Layout/index.js

```diff
@@ -0,0 +1,26 @@
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import * as actions from './LayoutActions';
+import reducer from './LayoutReducer';
+import { patternflyMenuItemsSelector } from './LayoutSelectors';
+
+import Layout from './Layout';
+
+// map state to props
+const mapStateToProps = state => ({
+  items: patternflyMenuItemsSelector(state),
+  isLoading: state.layout.isLoading,
```

Comment:
Could you use a selector for getting the `state.layout`?
We decided that we want single responsibility for selecting the relevant object from the state.